test/e2e/disk_encryption.go

```diff
@@ -0,0 +1,72 @@
+package e2e
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/Azure/ARO-RP/pkg/util/stringutils"
+	mgmtcompute "github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2020-06-01/compute"
+)
+
+var _ = Describe("Encryption at host should be enabled", func() {
+	BeforeEach(skipIfNotInDevelopmentEnv)
+
+	Specify("each VM should have encryption at host enabled", func() {
+		ctx := context.Background()
+
+		By("getting the resource group where the VM instances live in")
+		oc, err := clients.OpenshiftClustersv20200430.Get(ctx, vnetResourceGroup, clusterName)
+		Expect(err).NotTo(HaveOccurred())
+		clusterResourceGroup := stringutils.LastTokenByte(*oc.OpenShiftClusterProperties.ClusterProfile.ResourceGroupID, '/')
+
+		By("listing all VMs")
+		vms, err := clients.VirtualMachines.List(ctx, clusterResourceGroup)
+		Expect(err).NotTo(HaveOccurred())
+		Expect(vms).NotTo(HaveLen(0))
+
+		By("checking the encryption property on each VM")
+		for _, vm := range vms {
+			log.Println(vm.Name)
+			Expect(vm.SecurityProfile).To(Not(BeNil()))
+			Expect(vm.SecurityProfile.EncryptionAtHost).To(Not(BeNil()))
+			Expect(*vm.SecurityProfile.EncryptionAtHost).To(Equal(true))
+		}
+
+	})
+})
+
+var _ = Describe("Disk encryption at rest should be enabled with customer managed key", func() {
+	BeforeEach(skipIfNotInDevelopmentEnv)
+
+	// We have to get the disks by VM, because getting all disks by resource group does not work
```

Comment:
I think it is fine to check via VMs. But I'm wondering why you can't list disks by resoruce group?